### PR TITLE
Automate data update

### PR DESCRIPTION
**Pull request by clairezed**

### CODE REVIEW

**Comment by clairezed** [comment]
@DavidBruant There are two eslint errors I don't know how to manage. This is link to the way the Github wrapper script I used as a base is written, I guess. 

```
'User' is not defined. (no-undef) (500, 11)
'Repository' is not defined. (no-undef) (501, 17)
```

File: appscripts/githubApiWrapper.js [hunk 1]
```diff
@@ -0,0 +1,504 @@
...
+are permitted provided that the following conditions are met:
+
+- Redistributions of source code must retain the above copyright notice, this
+  list of conditions and the following disclaimer.
+- Redistributions in binary form must reproduce the above copyright notice, this
+  list of conditions and the following disclaimer in the documentation and/or
+  other materials provided with the distribution.
+- Neither the name "Development Seed" nor the names of its contributors may be
+  used to endorse or promote products derived from this software without
+  specific prior written permission.
```
Comment (DavidBruant) [inline appscripts/githubApiWrapper.js:19]: :+1:

File: appscripts/main.js [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+  try {
+    var resp = Github.Repository.createFile(branch, fileName, csvFile, 'Add file '+fileName);
+    console.log(resp);
+  } catch(e) {
+    var git_file_obj = Github.Repository.getContents({ref: branch}, fileName);
+    Github.Repository.updateFile(branch, fileName, csvFile, git_file_obj.sha, 'Update file '+fileName, {encode: false});
+  }
+}
+
+/* GREATLY inspired of https://gist.github.com/mderazon/9655893 */
```
Comment (DavidBruant) [inline appscripts/main.js:77]: :+1: I would have thought Google App Script would provide a function directly, but apparently they don't

File: appscripts/main.js [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+    .onFormSubmit()
+    .create();
+}
+
+function onFormSubmitTrigger(event) {
+  var address = formatAddress(event);
+  var response = UrlFetchApp.fetch('https://api-adresse.data.gouv.fr/search/?q='+address);
+  var coordinates = getAddressCoordinates(response);
+  setRangeCoordinates(event, coordinates);
+  commitDataToGithub();
```
Comment (DavidBruant) [inline appscripts/main.js:16]: :-)

File: appscripts/main.js [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+  return address;
+}
+
+function getAddressCoordinates(httpResponse) {
+  var data = httpResponse.getContentText('UTF-8');
+  var parsedData = JSON.parse(data);
+  if(parsedData.features[0]){
+    var coordinates = parsedData.features[0].geometry.coordinates;
+    return coordinates;
+  }else{
```
Comment (DavidBruant) [inline appscripts/main.js]: is this an indent style you chose?
Did you run eslint on this code?

**Review by DavidBruant — APPROVED** [review]
LGTM

**Pull request merged**